src/sentry/api/serializers/models/team.py

```diff
@@ -149,55 +194,24 @@ def serialize(
             "avatar": avatar,
         }
 
+        # Expandable attributes.
+        if self._expand("externalTeams"):
+            result["externalTeams"] = attrs["externalTeams"]
 
-class TeamWithProjectsSerializer(TeamSerializer):
-    def get_attrs(
-        self, item_list: Sequence[Team], user: Any, **kwargs: Any
-    ) -> MutableMapping[Team, MutableMapping[str, Any]]:
-        project_teams = list(
-            ProjectTeam.objects.filter(team__in=item_list, project__status=ProjectStatus.VISIBLE)
-            .order_by("project__name", "project__slug")
-            .select_related("project")
-        )
-
-        actor_mapping = {team.actor_id: team for team in item_list}
-        external_actors = list(ExternalActor.objects.filter(actor_id__in=actor_mapping.keys()))
+        if self._expand("organization"):
```

Comment:
@NisanthanNanthakumar `organization` expand is hidden in the diff.

Reply:
@mgaeta ahh I see, so do we need the `_collapse` helper method?

Reply:
since it looks like `self.collapse` will always be an empty list once it gets to the serializer